app/assets/stylesheets/_claims_importer.scss

```diff
@@ -10,28 +10,31 @@
   label {
     @include core-16;
   }
+
   .js-enabled & {
+
     input[type="file"],
     .no-file-selected [type="submit"] {
       @include visually-hidden;
     }
   }
+
   button.button-secondary {
     @include core-19;
   }
+
   .errors {
-    margin-top: 5px;
-    line-height: 1.2;
+    margin-top: $gutter-one-sixth;
     color: $red;
   }
 }
 
 .importer-results {
```

Comment:
i don't think this class is used. I think it was part of the json uploader/importer which has since been removed

Reply:
Thanks, this has been removed.